Podfile

```diff
@@ -21,6 +21,8 @@ end
 target 'podcasts' do
   platform :ios, app_ios_deployment_target.version
   common_pods
+  pod 'PulseCore', :git => 'https://github.com/kean/Pulse.git', :tag => '4.2.4', :configurations => ['Debug', 'Staging', 'Prototype']
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/f480240a65531a6607ddef00a3260a1d14d1a519/Podfile#L24"></span>Style/WordArray: Use <code>%w</code> or <code>%W</code> for an array of words.</td>
    </tr>
  </tbody>
</table>